roles/kubernetes/postinstall/files/renew-certs-kubeadm.timer

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Timer of Certificates rotation service for Kubernetes
+[Timer]
+OnCalendar=quarterly
```

Comment:
Running the renew command on all nodes, masters and nodes including, would cause a massive outage on the cluster when the kubelet restarts

Reply:
This Timer will only run on  master nodes because it are included with condition:
_when: inventory_hostname in groups['kube-master']_
In other words, on master nodes the certificates are rotated by kubeadm, but on the worker node them are rotated by collaboration with _kubelts_ and _kube-controller-manager_.

Reply:
Outage of cluster will not happen. Kubelets do the rotation without restart itself, and they aware about new certs immediately. But control-plane components can aware the certificate had changed after restart, I guess